### PR TITLE
Notify user that after enabling API they must clear cache to load new routes.

### PR DESCRIPTION
**Pull request by biozshock**
| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ x ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Issue(s) addressed                     | #10568 

#### Description:
While we wait for the decision on another PR that addresses the issue.
Add a note that users must clear cache after enabling API.
Please check wording.

#### Steps to test this PR:

1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. See the message is displayed on the API configuration page.

<a href="https://gitpod.io/#https://github.com/mautic/mautic/pull/11410"></a>

### CODE REVIEW

File: app/bundles/ApiBundle/Translations/en_US/messages.ini [hunk 1]
```diff
@@ -36,6 +36,7 @@ mautic.api.config.form.api.enabled="API enabled?"
 mautic.api.config.form.api.basic_auth_enabled="Enable HTTP basic auth?"
 mautic.api.config.form.api.basic_auth.tooltip="Enable HTTP basic authentication for Mautic's API. It is highly recommended to only use this with secure sites (HTTPS)."
 mautic.api.config.form.api.enabled.tooltip="Enable Mautic's API?"
+mautic.api.config.form.api.cache_notice="After enabling/disabling API you must clear cache using console command <code>bin/console cache:clear</code>."
```
Comment (RCheesley) [inline app/bundles/ApiBundle/Translations/en_US/messages.ini]:
```suggestion
mautic.api.config.form.api.cache_notice="After enabling/disabling the API you must clear your cache using the console command <code>bin/console cache:clear</code>."
```

**Review by RCheesley — CHANGES_REQUESTED** [review]
Thanks for the PR @biozshock, hopefully this will reduce the number of support questions!

Works as intended - please see some grammar suggestions.

screenshot-8080-biozshock-mautic-ekweucigm38 ws-eu63 gitpod io-2022 08 26-14_23_38

**Comment by nick-vanpraet** [comment]
image

Matches the post-grammar suggestions text, all good.

**Review by nick-vanpraet — APPROVED** [review]
Looks good to me

**Comment by kuzmany** [comment]
I am not sure If this message:

> After enabling/disabling the API you must clear your cache using the console command bin/console cache:clear

Is helpful for end users. Usually they don't have access to SSH and related know how.
Probably ask to you administrator for clear cache would be much better

**Comment by escopecz** [comment]
I'd like to avoid adding this alert and rather fix the issue. See https://github.com/mautic/mautic/pull/11420

**Comment by escopecz** [comment]
Closing in favore of https://github.com/mautic/mautic/pull/11420

**Pull request closed without merging**